src/Compilers/Core/Portable/Compilation/IOperation.cs

```diff
@@ -14,6 +14,12 @@ public interface IOperation
         /// Identifies the kind of the operation.
         /// </summary>
         OperationKind Kind { get; }
+
+        /// <summary>
+        /// Indicates whether the compiler has reported semantic errors about the operation.
+        /// </summary>
+        bool HasErrors { get; }
```

Comment:
The indicated errors can be either syntactic or semantic. The intent is to indicate that the individual operation is semantically unreliable. "DoNotTrustMe", "SomethingIsWrongWithMe", or "IsBad" would be accurate names, but I can almost guarantee that if I change the name someone else will want it brought into line with the name in the bound trees.